pkgs/data/misc/dns-root-data/default.nix

```
@@ -19,7 +19,8 @@ let
 in
 
 stdenv.mkDerivation {
-  name = "dns-root-data-2019-01-11";
+  pname = "dns-root-data";
+  version = "unstable-2019-01-11";
```

That date seems to be wrong.

```suggestion
  version = "2017-10-24";
```
According to the links that is the date and I would just use that without unstable.

https://gitlab.nic.cz/knot/knot-resolver/raw/cc5e14a264912/etc/root.hints
https://github.com/CZ-NIC/knot-resolver/blob/master/etc/root.hints

---

There are two independent pieces of data in there &ndash; IPs with names and DNSSEC root.  The DS set changed later: https://github.com/NixOS/nixpkgs/commit/c4a5565e7a77b1b6f217c2981099429942eab1a4

---

so i will revert this to `2019-01-11` without "unstable".